dvc/stage/__init__.py

```diff
@@ -273,7 +273,7 @@ def _read_env(self, out, checkpoint_func=None) -> Env:
             from dvc.output import Output
             from dvc.schema import LIVE_PROPS
 
-            env[DVCLIVE_PATH] = str(out.path_info)
+            env[DVCLIVE_PATH] = str(os.path.relpath(out.path_info, self.wdir))
```

Comment:
We usually use our own [`dvc.utils.relpath`](https://github.com/iterative/dvc/blob/0eda4bd91ceac9c5bfba30f52e7951b86ee7bfe9/dvc/utils/__init__.py#L314) which is already imported here.
```suggestion
            env[DVCLIVE_PATH] = str(relpath(out.path_info, self.wdir))
```